skrub/tests/test_interpolation_join.py

```diff
@@ -0,0 +1,117 @@
+import pandas as pd
+import pytest
+from sklearn.neighbors import KNeighborsRegressor
+
+from skrub import InterpolationJoin
+
+
+@pytest.fixture
+def buildings():
+    return pd.DataFrame(
+        {"latitude": [1.0, 2.0], "longitude": [1.0, 2.0], "n_stories": [3, 7]}
+    )
+
+
+@pytest.fixture
+def annual_avg_temp():
+    return pd.DataFrame(
+        {
+            "latitude": [1.2, 0.9, 1.9, 1.7, 5.0, 5.0],
+            "longitude": [0.8, 1.1, 1.8, 1.8, 5.0, 5.0],
+            "avg_temp": [10.0, 11.0, 15.0, 16.0, 20.0, None],
+        }
+    )
```

Comment:
Do we need the fixtures? We could define the dataframes at the top of the file and use them in the functions without passing them as parameters. scikit-learn does this a lot.

Reply:
sure, we can do that! one advantage of a fixture is we get a new dataframe each time. with a global variable all tests need to be careful not to modify it otherwise it can cause test failures that show up or not depending on the (unpredictable) order tests are executed in

Reply:
Ok got it, let's keep the fixtures then